### ISSUE TITLE
Storybook accessibility addon: accessibility issues detected

### ISSUE THREAD

**Issue by MattDClarke** [bryntum/support#10921]
When using the Bryntum React Grid with Storybook, the accessibility addon detects 4 accessibility violations:

![Storybook accessibility addon](https://github.com/MattDClarke/bryntum-grid-storybook/blob/main/src/assets/storybook-11ty-addon.png?raw=true)

You can run the example Storybook workshop using this [GitHub repo](https://github.com/MattDClarke/bryntum-grid-storybook). There are setup instructions in the README.

**Comment by matsbryntse** [comment]
Needs to be tested against 6.2 branch

**Comment by MattDClarke** [comment]
I edited the issue: I tested using the `nightly-next` version 6.2.20250310, I still get some accessibility issues.

**Comment by matsbryntse** [comment]
@MattDClarke Same? If different, can you please post screenshots?

**Comment by MattDClarke** [comment]
I get different accessibility issues.
The first screenshot above shows the 4 Grid accessibility issues when using Grid version 6.1.7.
The rest of the screenshots above show the 2 accessibility issues when using Grid version 6.2.20250310 or version 6.2.0-alpha-2
